res/css/photon/label.css

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.photon-label {
+  display: inline-flex;
```

Comment:
`.stackSettingsLabel` has also this, maybe we can remove it there? Although you may want to keep it explicit, up to you!